modules/ossm-deploy-single-tenant.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+[id="ossm-deploy-single-tenant_{context}"]
+= Single tenancy model
+
+[role="_abstract"]
+In Istio, a tenant is a group of users that share common access and privileges for a set of deployed workloads. Tenants can be used to provide a level of isolation between different teams.
+
+Single tenant, cluster-wide control plane configurations are deprecated as of {ProductName} version 1.0.  {ProductName} defaults to a multitenant model.
```

Comment:
Do we have documentation specific to versions prior to 1.0?

Reply:
Not really.